test/util/nightwatch-commands/axeCheck.js

```diff
@@ -19,16 +19,16 @@ export function command(context, config, _callback) {
   }, [axeSource]);
 
   // Run axe checks and report
-  this.executeAsync((innerContext, done) => {
+  this.executeAsync((innerContext, rules, done) => {
```

Comment:
@jbalboni Back to your question---I think you're saying you'd prefer to catch new regressions by changing individual `axeCheck` commands to be stricter, so that they will be merged into master and stay fixed. Then when we're ready to make _all_ the test strict, we just change the default rule set, and at our leisure update any `axeCheck` commands that unnecessarily specify the rules. 

Is that right? I think that's a fair point. Will go back and add that to the config parameter.